lambda/format-activity-log/models.ts

```diff
@@ -21,15 +23,22 @@ export interface Activity {
 }
 
 export interface ActivityLogEntry {
+  event_id: string;
   event_type: string;
   session_id: string;
   user_id: string;
   timestamp: number;
-  activities: Activity[];
-  truncated: boolean;
+  client_id: string;
+  reported_suspicious: boolean;
 }
 
 export interface UserActivityLog {
   txmaEvent: TxmaEvent;
   activityLogEntry: ActivityLogEntry | undefined;
```

Comment:
Can activityLogEntry ever be undefined now?

Reply:
Addressed , in this intance, UserActivityLog interface is not even needed at all, removed.